docs/appendices/release-notes/unreleased.rst

```diff
@@ -97,6 +97,9 @@ Changes
   where there are filters restricting the result set or if only a sub-set of
   the columns is selected.
 
+- Bump JNA library to version 5.6.0. This will make CrateDB start flawlessly
```

Comment:
As this will be backported and is treat as a fix, it must go into the `Fixes` section

Reply:
Thanks. I've amended the patch.